client/state/help/courses/reducer.js

```diff
@@ -9,12 +9,23 @@ import { combineReducers } from 'redux';
 import { createReducer } from 'state/utils';
 import {
 	HELP_COURSES_RECEIVE,
+	HELP_COURSES_REQUEST,
+	HELP_COURSES_REQUEST_FAILURE,
+	SERIALIZE,
 } from 'state/action-types';
 
+export const requesting = createReducer( false, {
+	[ HELP_COURSES_REQUEST ]: () => true,
+	[ HELP_COURSES_RECEIVE ]: () => false,
+	[ HELP_COURSES_REQUEST_FAILURE ]: () => false,
+	[ SERIALIZE ]: () => false,
+} );
+
 export const items = createReducer( null, {
 	[ HELP_COURSES_RECEIVE ]: ( state, { courses } ) => courses
 } );
```

Comment:
It would be good to have a schema to validate the data we're getting from the server and capture it as documentation. As a bonus, adding the schema as the last parameter in this `createReducer` helper will persist the data across page reloads. [`state/plans`](https://github.com/Automattic/wp-calypso/tree/master/client/state/plans) makes a great example